docs/UserGuide.md

```diff
@@ -326,6 +326,36 @@ Okay, IBM has been marked as acted upon!
 In this example, the first instrument is `IBM` so it has been
 checked as complete.
 
+### Editing an instrument: `edit`
+Edit an existing instrument parameters.
+
+**Format**
+```
+mTracker$> edit INDEX
+```
+**Example usage**
+
+```
+mTracker$> edit 2
+```
+
+**Expected outcome:**
+
+Upon entering the `edit` command, mTracker prompts for the parameters of 
+instrument to be edited:
+
+```
+mTracker$> edit 2
+	Please enter one of more of TYPE parameters to edits:
```

Comment:
the "TYPE" was meant to be a placeholder, in your specific example it should be stock